docs/Configuration.md

```diff
@@ -27,14 +27,6 @@ Path to target binary.
 target_path: path/to/target/binary
 ```
 
-### elf_path
-
-Path to the final elf target
-
-#### Default
-Path to the binary that was used as the input to `create_config.py`
-
-
```

Comment:
Why did you remove this?

Reply:
I just realized my mistake. I thought it was not used anymore as I couldn't find references to it throughout the project, but upon searching a few minutes ago I see that it put into the `ld` script. Let me fix my mistake